drivers/mtd/mtd_progmem.c

```diff
@@ -371,7 +371,6 @@ static int progmem_ioctl(FAR struct mtd_dev_s *dev, int cmd,
       case MTDIOC_ERASESTATE:
         {
           FAR uint8_t *result = (FAR uint8_t *)arg;
-
```

Comment:
why change?

Reply:
Just sync style across multiple files:
https://github.com/apache/incubator-nuttx/blob/af72376773aa1434fe247ccb7c578abea9373fe7/drivers/mtd/rammtd.c#L430..L436
https://github.com/apache/incubator-nuttx/blob/af72376773aa1434fe247ccb7c578abea9373fe7/drivers/mtd/filemtd.c#L464..L470
https://github.com/apache/incubator-nuttx/blob/a59aae69270f547d976651da82543daccaa4bb3c/arch/risc-v/src/esp32c3/esp32c3_spiflash.c#L889..896
and many others